guides/content/release_notes/3_1_0.md

```diff
@@ -127,4 +127,8 @@ You can view the full changes using [Github Compare](https://github.com/spree/sp
   changed easily by adding the `path: Spree.admin_path` option in the routes.
 
     [Rick Blommers](https://github.com/spree/spree/pull/6739)
+
+* Change to Use Time.current instead of Time.now
```

Comment:
Could you also please explain how this uses the application timezone rather than default utc. Idea being for those who don't know the difference we should explain the change in behavior.